src/Menu/Menu.tsx

```diff
@@ -63,6 +64,8 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(
         fullScreenOnMobile={fullScreenOnMobile}
         ref={ref}
         setStyle={setStyle}
+        withOverlay={withOverlay}
+        simulated
```

Comment:
Why do you need `simulated` ?

Reply:
I need to pass parent background to child.